tests/namespace/test_mibs.py

```diff
@@ -36,6 +36,17 @@ def test_init_namespace_sync_d_lag_tables(self):
         self.assertTrue("PortChannel_Temp" in lag_name_if_name_map)
         self.assertTrue(lag_name_if_name_map["PortChannel_Temp"] == [])
 
+    def test_init_sync_d_interface_tables_for_recirc_ports(self):
```

Comment:
thank you, will be good to add test case to verify the oid output like in test_interfaces/test_pfc/test_queues

Reply:
@SuvarnaMeenakshi I extended test to verify the oids of recirc ports.

Reply:
https://github.com/Azure/sonic-snmpagent/issues/256 - created this issue to track this.